readme.md

```diff
@@ -416,21 +413,25 @@ post '/instructors' do
 end
 ```
 
+<!-- AM: What's the best way to demo before/after namespacing? -->
 
```

Comment:
I guess the question is more of how to display it, since you can't print it to the server log. I guess I'll just print it on the page or something.